script/hassfest/translations.py

```diff
@@ -368,6 +368,9 @@ def gen_strings_schema(config: Config, integration: Integration) -> vol.Schema:
                             },
                             slug_validator=translation_key_validator,
                         ),
+                        vol.Optional(
+                            "native_unit_of_measurement"
```

Comment:
Translations for units, will only happen for custom units of measurements that do not fit our provided units. As a result, it will only be used for units without conversion, meaning it will always be the native unit.

With that in mind, we could just as well just rename this to `unit_of_measurement`.

```suggestion
                            "unit_of_measurement"
```

../Frenck